docs/search_relevance/query_design.md

```diff
@@ -28,11 +28,68 @@ Each query has:
 
 ## Intentions & Expectations
 
+### Phrase matching
+
+| Status | Query name | Ranking evaluation test |
+| :--- | :--- | :--- |
+| TODO | PhraseMatchQuery | TBD |
+
+#### Intentions
+
+Searching for an exact, ordered set of tokens, in quotation marks
```

Comment:
Another PR: have a glossary, add `tokens`.

Reply:
https://github.com/wellcomecollection/catalogue/issues/430

Reply:
Or better - `Searching for a phrase`

### glossary 
- phrase: set of ordered tokens
- token: word (<- or whatever)